vnext/just-task.js

```diff
@@ -36,6 +36,12 @@ task('apiDocumenter', () => {
   );
 });
 
+task('codegen', () => {
+  execSync(
+    'npx react-native-windows-codegen --files Libraries/**/Native*.js --namespace Microsoft::ReactNativeSpecs',
```

Comment:
Out of curiosity, how long does this end up taking?

Reply:
< 1sec